Makefile

```diff
@@ -918,14 +918,32 @@ lockfile-check:
 		exit 1; \
 	fi
 
+LOCALE_DIR := options/locale
+LOCALE_FILES := $(wildcard $(LOCALE_DIR)/*.json)
+
+.PHONY: translation-check
+translation-check:
+	npm install -g jsonlint find-duplicated-property-keys
```

Comment:
Add them to devDependencies:

```
pnpm install - D jsonlint find-duplicated-property-keys
```

And  then use them as this:

```
$(NODE_VARS) pnpm exec jsonlint
$(NODE_VARS) pnpm exec find-duplicated-property-keys
```

Reply:
d7702ead0e